internal/chainlink/chainlink_service.go

```diff
@@ -57,10 +52,7 @@ func (service chainlinkService) Get(name string) (*chainlinkv1alpha1.Node, *erro
 // Create creates chainlink node from the given spec
 func (service chainlinkService) Create(dto *ChainlinkDto) (*chainlinkv1alpha1.Node, *errors.RestErr) {
 	node := &chainlinkv1alpha1.Node{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      dto.Name,
-			Namespace: "default",
-		},
+		ObjectMeta: dto.ObjectMetaFromNamespaceDto(),
```

Comment:
This way, all created chainlink nodes will use the workspace name.
Use the node name `dto.Name` instead.